microsoft-edge/web-platform/user-agent-string.md

```diff
@@ -9,24 +9,112 @@ ms.prod: microsoft-edge
+-	*Client-side*: UA Client Hints can be accessed via JavaScript by querying the value of `navigator.userAgentData`
+
+**User Agent String**
+-	*Server-side*: UA strings are sent by default with the User-Agent `HTTP` header 
+-	*Client-side*: UA strings can be accessed via JavaScript by querying the value of `navigator.userAgent`
 
 Microsoft recommends that web developers make use of [feature detection](https://developer.mozilla.org/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection) whenever possible to improve code maintainability, reduce code fragility, and eliminate the risk of code breakage in the event of future UA string updates.  
 
 For cases where feature detection is not applicable and UA detection must be used, the format of the Microsoft Edge UA on desktop is as follows:
 
+## User Agent Client Hints
+Starting from Edge90 Stable, developers can access browser information in a cleaner, more privacy preserving way. 
+
+### UA-CH HTTPS Header
+
+By default, Chromium browsers like Edge will send the `Accept-CH-UA` request header in the following format: 
+
+```HTTP
+Sec-CH-UA: “Chromium”;v=”91”, “Microsoft Edge”;v=”91”,”Dummy;Browser Brand”;v=”99”
+Sec-CH-UA-Mobile: ?0
+```
+Note: Client Hints are only sent over `HTTPS`. 
+
+More information, if needed, can be accessed by sending subsequent requests with the desired headers: 
+
+#### User-Agent Request & Response Headers
+| Response Header | Example Response Value |
+| ----------- | ----------- |
+|`Sec-CH-UA`|`“Chromium”;v=”91”, “Microsoft Edge”;v=”91”,”GREASE”;v=”99"`|
+|`Sec-CH-UA-Mobile`|`false`|
+|`Sec-CH-UA-Full-Version`|`91.0.866.0`|
+|`Sec-CH-UA-Platform`|`Windows`|
+|`Sec-CH-UA Platform-Version`|`10.0`|
+|`Sec-CH-UA-Arch`|`x86`|
+|`Sec-CH-UA-Model`||
+
+### UA-CH JavaScript API
+
+The default return value from `navigator.userAgentData` is in the following format: 
+
+```javascript
+{brands: 
+    [{brand: “Chromium”,”version”:”91”}, 
+    brand: “Microsoft Edge”,”version”:”91”}, 
+    brand: “GREASE”,”version”:”99”},]
+mobile: false
+}
+```
+More detailed information, if needed, can be accessed with [getHighEntropyValues()](https://wicg.github.io/ua-client-hints/#getHighEntropyValues), for example, calling
+
+```javascript
+navigator.userAgentData.getHighEntropyValues(
+    ["architecture", "model", "platform", "platformVersion", "uaFullVersion"])
+    .then(ua => { console.log(ua) });
+```
+
+May output the following: 
+
+```javascript
+{architecture: "x86", 
+model: "", 
+platform: "Windows", 
+platformVersion: "10.0", 
+uaFullVersion: "92.0.866.0"}
+```
+
+## Best Practices for Using User Agent Client Hints
+Since the order brands appear in may change over time, developers are discouraged from hard-coding static checks for values in specific positions. To further prevent abuse of the brands list, GREASE values may also be sent, consisting of `null`, no value, or random values.

```

Comment:
Recommended wording:

Since the set of brands and the order they appear in will change over time, developers should never hard-code indices into the array of returned brands. To help ensure developers spot such bugs in their sites early, the browser includes a "GREASE" brand value which will change over time and be formatted in such a way as to trip up on common string parsing bugs.

Reply:
Updated.